back-end/user.js

```diff
@@ -0,0 +1,28 @@
+/*
+This is the user setting page. I want to return a json object here with
+the user's data
+*/
+
+//The constant variables we'll need
+const express = require('express')
+const app = express()
```

Comment:
@RyanmLin is this necessary for this file? The current purpose seems to be the test2 function which do not use the express framework so why is it called here?